### ISSUE TITLE
 Predict fuel efficiency: regression example should not normalize one-hot values

### ISSUE THREAD

**Issue by richeyd** [tensorflow/tensorflow#24342]
**Describe the current behavior**
In the Predict fuel efficiency: regression lesson (TensorFlow->Learn->Tutorials->Learn and use ML->Regression), the normalize method is called to normalize the entire dataset, including the one-hot columns.

**Describe the expected behavior**
The one-hot columns in the dataset should not be normalized.

**Code to reproduce the issue**
dataset['USA'] = (origin == 1)*1.0
dataset['Europe'] = (origin == 2)*1.0
dataset['Japan'] = (origin == 3)*1.0
train_dataset = dataset.sample(frac=0.8,random_state=0)
train_stats = train_dataset.describe()
train_stats.pop("MPG")
train_stats = train_stats.transpose()

def norm(x):
  return (x - train_stats['mean']) / train_stats['std']
normed_train_data = norm(train_dataset)

**Comment by harshini-gadige** [comment]
@lamberta  Could you please check if the normalization to be removed for one hot columns ?

**Comment by yashk2810** [comment]
Thanks for reporting. Can you send a PR here (https://github.com/tensorflow/docs/blob/master/site/en/tutorials/keras/regression.ipynb) to fix it.

**Comment by rmothukuru** [comment]
@richeyd,
In the latest [Predict Fuel Effiency : Regression Example](https://www.tensorflow.org/tutorials/keras/regression), the **`One-Hot Values`** are no more normalized. Can you please confirm if we can close this issue? Thanks!

**Comment by MarkDaoust** [comment]
>  the One-Hot Values are no more normalized. 

This is the line of interest, where it adapts the normalization layer to all the features.

```
normalizer.adapt(np.array(train_features))
```

> should not normalize one-hot values

I would say "they don't need to be normalized.". But It doesn't do any harm, and it keeps this tutorial **simple**.

I think an apropriate fix here would be to add a `Note:` paragraph that to section, and include links to these:

1. https://www.tensorflow.org/tutorials/structured_data/preprocessing_layers
2. https://www.tensorflow.org/guide/keras/preprocessing_layers

**Comment by rmothukuru** [comment]
@MarkDaoust,
Sure, I can add that Fix.

**Comment by rmothukuru** [comment]
Closing the issue as the PR is merged.
